mbed-trace/mbed_trace.h

```diff
@@ -81,48 +81,62 @@ extern "C" {
     and 5 lower bits are trace level configuration */
 
 /** Config mask */
-#define TRACE_MASK_CONFIG         0xE0
+#define TRACE_MASK_CONFIG         0xF0
 /** Trace level mask */
-#define TRACE_MASK_LEVEL          0x1F
+#define TRACE_MASK_LEVEL          0x0F
 
 /** plain trace data instead of "headers" */
-#define TRACE_MODE_PLAIN          0x80
+#define TRACE_MODE_PLAIN          0x40
 /** color mode */
-#define TRACE_MODE_COLOR          0x40
+#define TRACE_MODE_COLOR          0x20
 /** Use print CR before trace line */
-#define TRACE_CARRIAGE_RETURN     0x20
-
-/** used to activate all trace levels */
-#define TRACE_ACTIVE_LEVEL_ALL    0x1F
-/** print all traces same as above */
-#define TRACE_ACTIVE_LEVEL_DEBUG  0x1f
-/** print info,warn and error traces */
-#define TRACE_ACTIVE_LEVEL_INFO   0x0f
-/** print warn and error traces */
-#define TRACE_ACTIVE_LEVEL_WARN   0x07
-/** print only error trace */
-#define TRACE_ACTIVE_LEVEL_ERROR  0x03
-/** print only cmd line data */
-#define TRACE_ACTIVE_LEVEL_CMD    0x01
-/** trace nothing  */
-#define TRACE_ACTIVE_LEVEL_NONE   0x00
+#define TRACE_CARRIAGE_RETURN     0x10
 
+/** this print is some silly deep information for debug purpose */
+#define TRACE_LEVEL_SILLY         0x07
```

Comment:
Should it be just called DEEP rather than SILLY?

Reply:
I'm fine with both keys. I just pick `silly` keyword from [winston](https://github.com/winstonjs/winston#logging) library :) Votes ?